pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -826,13 +823,25 @@ func markObjectAsVerifying(ctx *verifierContext,
 	// in DownloadDirname/<objType>/verifier/. Form a shorter name for
 	// DownloadDirname/<objType/>verified/.
 
+	var pendingDirname string
+	var verifierDirname string
+	var pendingFilename string
+	var verifierFilename string
 	objType := status.ObjType
 	downloadDirname := types.DownloadDirname + "/" + objType
-	pendingDirname := downloadDirname + "/pending/" + status.ImageSha256
-	verifierDirname := downloadDirname + "/verifier/" + status.ImageSha256
+	if status.IsContainer {
```

Comment:
We seem to be repeating the logic .. Can we have function to return these values that can be called from both places?